### PR TITLE
adding easy problems in python

### PR DESCRIPTION
**Pull request by mehak-sachdeva**
Easy problems done until today
@aus2101

### CODE REVIEW

File: Test/NotCSharp/Python/EASY-SquaresOfASortedArray.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#https://leetcode.com/problems/squares-of-a-sorted-array/
+
+class Solution:
+    def sortedSquares(self, A: List[int]) -> List[int]:
+        l = 0
+        r = len(A)-1
+        lst=[]
+
+        while l<=r:
```
Comment (aus2101) [inline Test/NotCSharp/Python/EASY-SquaresOfASortedArray.py]: indentation error on while

File: Test/NotCSharp/Python/EASY-FindWordsThatCanBeFormedByCharacters.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+                    break
+                elif word.count(ch)>char_dic[ch]:
+                    check = False
+                    break
+                else:
+                    count +=1
+            if check:
+                tot+=count
+
+        return tot
```
Comment (aus2101) [inline Test/NotCSharp/Python/EASY-FindWordsThatCanBeFormedByCharacters.py]: can you double check, I think there's an indentation error on the last line

File: Test/NotCSharp/Python/EASY-SquaresOfASortedArray.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+            if (A[l]**2<A[r]**2):
+                lst.append(A[r]**2)
+                r-=1
+            else:
+                lst.append(A[l]**2)
+                l+=1
+        return lst[::-1]
```
Comment (aus2101) [inline Test/NotCSharp/Python/EASY-SquaresOfASortedArray.py]: can try something like this
asq = [x*x for x in A]
return sorted(asq)

File: Test/NotCSharp/Python/EASY-UniqueMorseCodeWords.py [hunk 4]
```diff
@@ -0,0 +1,14 @@
+#https://leetcode.com/problems/unique-morse-code-words/
+
+class Solution:
+    def uniqueMorseRepresentations(self, words: List[str]) -> int:
+        morse_lst = []
+        morse_dic = [".-","-...","-.-.","-..",".","..-.","--.","....","..",".---","-.-",".-..","--","-.","---",".--.","--.-",".-.","...","-","..-","...-",".--","-..-","-.--","--.."]
+
+        for word in words:
```
Comment (aus2101) [inline Test/NotCSharp/Python/EASY-UniqueMorseCodeWords.py]: indentation error in for loop

Comment (mehak-sachdeva) [reply Test/NotCSharp/Python/EASY-UniqueMorseCodeWords.py]: Sorry, I am not seeing this on my end.

File: Test/NotCSharp/Python/EASY-FindWordsThatCanBeFormedByCharacters.py (hunk 2 above)
Comment (mehak-sachdeva) [reply Test/NotCSharp/Python/EASY-FindWordsThatCanBeFormedByCharacters.py]: No indentation error on my end,

**Pull request merged**